lib/rules/one-var.js

```diff
@@ -62,20 +120,46 @@ module.exports = function(context) {
         "FunctionDeclaration": startFunction,
         "FunctionExpression": startFunction,
         "ArrowFunctionExpression": startFunction,
+        "BlockStatement": startBlock,
```

Comment:
hmmmm… I just added the following tests without adding either of those to the code, and everything still passes.

```js
// valid
        {
            code: "let foo = true; for (let i = 0; i < 1; i++) { let foo = false; }",
            ecmaFeatures: {
                blockBindings: true
            },
            args: [2, {var: "always", let: "never", const: "never"}]
        },
        {
            code: "let foo = true; switch(foo) { case true: let bar = 2; break; case false: let bar = 3; break; }",
            ecmaFeatures: {
                blockBindings: true
            },
            args: [2, {var: "always", let: "never", const: "never"}]
        }
```

Did I make some dumb mistake in the tests?